x-pack/test/accessibility/apps/group1/helpers.ts

```diff
@@ -14,7 +14,11 @@ export async function deleteAllPipelines(client: Client, logger: any) {
   await logger.debug(pipelines);
   if (pipeLineIds.length > 0) {
     await asyncForEach(pipeLineIds, async (newId) => {
-      await client.ingest.deletePipeline({ id: newId });
+      try {
+        await client.ingest.deletePipeline({ id: newId });
+      } catch (error) {
+        await logger.debug(`Error deleting pipeline: ${newId}`);
+      }
```

Comment:
The pipeline `.edr-workflow-insights-ingest-pipeline` is set as the default for the index `.edr-workflow-insights-default`. Attempting to remove the pipeline without removing the index causes an error. I’ve opted to simply catch the error instead of implementing case-by-case handling to keep things simpler. Let me know if you think a more precise approach would be better.

Reply:
I think should be fine, would you mind leaving a comment explaining the reasoning in there for posterity?

Reply:
@sabarasaba 
Agree, added with https://github.com/elastic/kibana/pull/204242/commits/71cc84661701f87da22f292fe0b2efd57f665781